readthedocs/projects/migrations/0116_remove_old_issues.py

```diff
@@ -0,0 +1,103 @@
+        ),
+        migrations.RemoveField(
+            model_name="historicalproject",
+            name="enable_pdf_build",
+        ),
+        migrations.RemoveField(
+            model_name="historicalproject",
+            name="install_project",
+        ),
+        migrations.RemoveField(
+            model_name="historicalproject",
+            name="path",
+        ),
+        migrations.RemoveField(
+            model_name="historicalproject",
+            name="python_interpreter",
+        ),
+        migrations.RemoveField(
+            model_name="historicalproject",
+            name="use_system_packages",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="conf_py_file",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="enable_epub_build",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="enable_pdf_build",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="install_project",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="path",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="python_interpreter",
+        ),
+        migrations.RemoveField(
+            model_name="project",
+            name="use_system_packages",
+        ),
+        migrations.AlterField(
+            model_name="historicalproject",
+            name="documentation_type",
+            field=models.CharField(
+                choices=[
+                    ("sphinx", "Sphinx Html"),
+                    ("mkdocs", "Mkdocs"),
+                    ("sphinx_htmldir", "Sphinx HtmlDir"),
+                    ("sphinx_singlehtml", "Sphinx Single Page HTML"),
+                ],
+                default="sphinx",
+                help_text='Type of documentation you are building. <a href="http://www.sphinx-doc.org/en/stable/builders.html#sphinx.builders.html.DirectoryHTMLBuilder">More info on sphinx builders</a>.',
+                max_length=20,
+                verbose_name="Documentation type",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="project",
+            name="documentation_type",
+            field=models.CharField(
+                choices=[
+                    ("sphinx", "Sphinx Html"),
+                    ("mkdocs", "Mkdocs"),
+                    ("sphinx_htmldir", "Sphinx HtmlDir"),
+                    ("sphinx_singlehtml", "Sphinx Single Page HTML"),
+                ],
+                default="sphinx",
+                help_text='Type of documentation you are building. <a href="http://www.sphinx-doc.org/en/stable/builders.html#sphinx.builders.html.DirectoryHTMLBuilder">More info on sphinx builders</a>.',
+                max_length=20,
+                verbose_name="Documentation type",
+            ),
+        ),
```

Comment:
Seems like this field is being set a non-null again here instead of being deleted.

Reply:
Hrm... Not sure what happened here. I will try to re-create this migration again.

Reply:
I re-ran the `makemigrations` command and it's seems that Django detects this as an `AlterField` for some reason on both models.

Reply:
I found the issue. The field is duplicated. It's also present in https://github.com/readthedocs/readthedocs.org/blob/5c556ec4bc04d3566ce88cace101f330ea48ca41/readthedocs/projects/models.py#L315-L325

Reply:
Actually, it seems this field is being used in other places as well. So, I will keep it for now and focus on the other ones. We can remove it later.

Reply:
BTW, this was a _great_ finding! Thanks for this review 👍🏼